### PR TITLE
Version scheme

### PR DESCRIPTION
**Pull request by benjaoming**
During the release of Kolibri 0.9, I accidentally bumped the Perseus plugin from 0.8 to 0.9 thinking that they were supposed to be aligned :)

It was actually called 0.9.0, like the Kolibri versioning scheme, with the little extra `.0`.

So when people were chatting about the release and Kolibri vs. kolibri-exercise-perseus-plugin, things just got really confusing and had to be like "oh which release are you talking about?" :)

Have also simplified instructions and added a `check-build` command for the `Makefile`.

TODO:

* [x] Bumps version to 1.1
* [x] Add release instructions in command line like Kolibri
* [x] Activate `make check-build` for `make dist` ~and make something that checks for translation files as well~ (didn't know how to achieve this easily)

Also a TODO: Automate the downloading of translation files somehow.

I also don't get this @rtibbles : To build the distribution, do I have to run `yarn install` from Kolibri with a linked editable version of kolibri-exercise-perseus-plugin ? In that case, I would like to add an extra assertion in `check-build` so we can verify if necessary files are created or recently updated.

### CODE REVIEW

**Comment by benjaoming** [comment]
@rtibbles @indirectlylit this is ready for review following up on the release process we just went through.

**Review by indirectlylit — APPROVED** [review]
haven't tested but these changes all look reasonable to me

**Comment by indirectlylit** [comment]
whoops, target branch is incorrect - should be develop

File: README.rst [hunk 1]
```diff
@@ -20,70 +21,114 @@ Khan Academy's exercise question editor and renderer.
...
+    yarn install
 
-5. Finally, to copy over Mathjax into the static folder, run the following command (you will need to do this if you update the version of Perseus in the repo also):
+   Make sure that this message **DOES NOT** appear in your logging output of
+   ``yarn install``, it means that you are not using the development version::
+
+    WARNING  assets/src/module.js not found for plugin exercise_perseus_render_module.
+
+5. After cloning for the first time, or if you made any changes to the project,
+   you need to rebuild it::
```
Comment (rtibbles) [inline README.rst:74]: This might be a misleading description? Perhaps 'you need to re-copy the static assets from the perseus repo'?

**Pull request merged**